frontend-mini-challenge/components/LoggedInPage.tsx

```diff
@@ -131,13 +153,15 @@ Then when you make use of this page component:
     </Page>
 
 */
-export async function getServerSidePropsForLoggedInPage({ query, req }) {
-  const whoAmIResponse = await serverSideWhoAmI({ query, req });
+
+
+export const getServerSidePropsForLoggedInPage = async ({ query, req }: GetServerSidePropsContext) => {
```

Comment:
I have mentioned this one multiple times now. If you do not understand a comment then please do not ignore the comment.  I really don't like repeating myself this many times.

Look at how much information is inside GetServerSideProps: 
https://github.com/vercel/next.js/blob/v10.2.0/packages/next/types/index.d.ts#L136-L151

if 90% of the interface is completely inaccurate then it's a bad choice. 

Please use something appropriate here.  GetServerSidePropsContext is inappropriate.

If you disagree then say so. Do not simply ignore this feedback.